aas_core_meta/v3_1.py

```diff
@@ -1394,8 +1352,8 @@ class Path_type(Identifier, DBC):
 
     .. note::
 
-        Any string conformant to RFC8089 , the “file” URI scheme (for
-        relative and absolute file paths)
+        Any string conformant to RFC 2396 and amended by RFC 2732, the “file” URI scheme
+        for XSL Schema 1.0 relative and absolute file paths
```

Comment:
```suggestion
        for XSD 1.0 relative and absolute file paths
```

I suppose XSL is a typo?